### ISSUE TITLE
Please support macOS Mojave dark mode

### ISSUE THREAD

**Issue by singularitti** [Gisto/Gisto#247]
Since macOS 10.14 (Mojave) has been released, which provides a [Dark mode](https://developer.apple.com/design/human-interface-guidelines/macos/visual-design/dark-mode/), would you consider support the Dark mode? For example, a Dark title bar, status bar, preferences pane, and so on?

Please have a look at [this page](https://developer.apple.com/documentation/appkit/supporting_dark_mode_in_your_interface).

**Comment by sanusart** [comment]
There will be partial support starting from [v1.11.1-beta.0](https://github.com/Gisto/Gisto/releases/tag/v1.11.1-beta.0)

TODO:
- Theme itself still need to be adjusted and refactored.
- Auto switch theme to match dark

At the moment, this what can be achieved in dark mode:
![image](https://user-images.githubusercontent.com/267718/55031682-c6208280-5017-11e9-9062-f20c39ef018d.png)

After switching to Appearance -> dark, app color can be picked from settings -> theme to match color of dark mode.

**Comment by sanusart** [comment]
I'll reopen as work still to be done.

**Comment by singularitti** [comment]
Great! Thank you very much!

Just have a little more request:
<img width="1201" alt="bar" src="https://user-images.githubusercontent.com/25192197/55034571-197cdb80-4fec-11e9-805f-1c25ae1792f3.png">
Is it possible to adjust the color of this bar to be gray, or dark? It stands out against the dark background.

**Comment by sanusart** [comment]
@singularitti 
Yes. That is why I've left the issue open, in order to gradually adjust the theme to support dark mode. It need way more changes than I can handle in short time.

**Comment by sanusart** [comment]
Playing around with different ui mock https://unruffled-lumiere-1d31d8.netlify.com/ 
Let's see where it goes.

**Comment by singularitti** [comment]
I do not exactly know how to use it but the color looks good.

**Comment by sanusart** [comment]
Just playing with it. Not sure gonna use it in the real app just yet.

**Comment by sanusart** [comment]
Closing as new rewritten version upcoming 

Ref: https://github.com/Gisto/Gisto/issues/387
